myrater/external_rater.py

```diff
@@ -0,0 +1,110 @@
+#!/usr/bin/env python
+import argparse
+import sys
+import json
+from socotratools.client import SocotraClient
+import myrater
+
+
+# The external rater only gives us locators and the policy, but we actually
+# want the full characteristic and the full exposure and peril from the
+# policy to do the appropriate calculations.  This method returns the 
+# characteristic and parent object.
```

Comment:
This description is confusing to me. Can you please reword, e.g., "external rater provides locators" What locators? Policy? External rater object? Maybe this information is in APIs or Feature documentation and you can link if you don't want to repeat?